python/src/energuide/extracted_datatypes.py

```diff
@@ -105,16 +106,16 @@ class _Door(typing.NamedTuple):
 
 class _Window(typing.NamedTuple):
     label: str
-    glazing_types_english: typing.Optional[str]
-    glazing_types_french: typing.Optional[str]
-    coatings_tints_english: typing.Optional[str]
-    coatings_tints_french: typing.Optional[str]
+    glazing_type_english: typing.Optional[str]
+    glazing_type_french: typing.Optional[str]
+    coating_tint_english: typing.Optional[str]
+    coating_tint_french: typing.Optional[str]
```

Comment:
I don't mind since it reads better this way, but the original had the `coatings_tints_english` had the words as plural because that's what they are in the xml
```
<CoatingsTints code="3">
    <English>Low-E .20 (hard1)</English>
    <French>Faible E .20 (Dur 1)</French>
</CoatingsTints>
```

Reply:
I'll change it back then.

Reply:
Actually no, I don't think I will. Let's leave it as-is.